docs/configure-custom-values-envoy.md

```diff
@@ -21,25 +21,19 @@ For example, if you want to configure the Scalar Envoy for ScalarDB Server, you
 
 ## Required configurations
 
-### Image configurations
+### Service configurations
+
+You must set `envoy.service.type` to specify the Service resource type of Kubernetes.
 
-You must set `envoy.image.repository` and `envoy.image.version`. Please specify the container repository information that you pull the Scalar Envoy container image.
+If you assume that you accept client requests from inside of the Kubernetes cluster only (i.e., you deploy your client applications on the same Kubernetes cluster as Scalar products), you can set `envoy.service.type` to `ClusterIP`. This configuration doesn't create any load balancers provided by cloud providers.
```

Comment:
I separated the `Service` configuration into `ClusterIP (accept client requests from inside of the Kubernetes cluster)` and `LoadBalancer (accept client requests from outside of the Kubernetes cluster)`.